config/application.rb

```diff
@@ -140,6 +140,8 @@ class Application < Rails::Application
 
     # Set to true to enable managing different tenants using the same application
     config.multitenancy = Rails.application.secrets.multitenancy
+
```

Comment:
Should we add a comment to the `config.devise_lockable` like we do for `config.multitenancy`? Should we use alphabetical order in the custom application config options, we are adding?

Reply:
cc @taitus